api/build.gradle.kts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2023 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+  `java-library`
+  `maven-publish`
+  BuildSupport
+}
+
+dependencies {
+  implementation(libs.guava)
+  implementation(libs.slf4j)
+  implementation(libs.logback.classic)
```

Comment:
Logback is definitely _not_ necessary.

Reply:
Last time when I checked, Logs were not printing for me with just `slf4j` depedency and `logback.xml` config. 
Then I added this dependency as suggested by the article  https://www.baeldung.com/logback#classpath
and it worked. 

Let me recheck it without it.

Reply:
`logback.classic` is still needed (as runtime dependency). Removed `logback.core`